sympy/functions/elementary/hyperbolic.py

```diff
@@ -73,7 +73,14 @@ class sinh(HyperbolicFunction):
     r"""
     The hyperbolic sine function, `\frac{e^x - e^{-x}}{2}`.
 
-    * sinh(x) -> Returns the hyperbolic sine of x
+    * sinh(x) -> Returns the hyperbolic sine of x.
```

Comment:
Let's write these as plain English sentences, like 
```suggestion
    sinh(x) is the hyperbolic sine of x.
```

Reply:
Changed for all the classes.